### ISSUE TITLE
Installation troubles when there's not enough space on startup volume

### ISSUE THREAD

**Issue by tempelmann** [utmapp/mac.getutm.app#23]
TL;DR – User needs to be asked where to store the initial VM, especially if the startup disk's space is not sufficient. Also, when moving a freshly created VM, it loses its Boot IPSW image.

I am trying to install the Ventura beta (supposedly, others had success with it) on a MB Air M1.

My M1 Mac's disk layout is that I have a fairly small partition for just the macOS system, and then symlink most of the home folders (Desktop, Downloads, Documents) to a different volume that has lots of space. The Library folder remains private to the system partition, though.

The problem with the UTM installation is that it does not ask me where I want to store the VM initially. It always places it in its Sandbox container's Documents folder, which is on the startup disk. In my case, that's too small. So, if I start the VM, it'll soon tell me that something failed, probably because it ran out of disk space.

So I retried: I create a new macOS VM with the IPSW, and before starting it, I use the UTM UI to move the VM to another volume. That appears to work, but then I find that it has lost the IPSW reference that I had specified during the VM setup, because when I start the VM now, it'll open a large black screen and nothing happens, not even an error, and no prompt whether I want to install macOS.

So I re-add the boot img (IPSW) in the settings for the VM and start it again. This time, I get prompted to confirm the macOS intall again.

But still, I get the error "Internal Virtualization error. Installation failed." eventually. Not sure what that is about. Possibly an unrelated issue.
